<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optionsDialog</class>
 <widget class="QDialog" name="optionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/48x48.png</normaloff>:/48x48.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QFrame" name="verticalFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>721</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="layout" stretch="0">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QTabWidget" name="SystemResouces">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>General</string>
         </attribute>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>260</y>
            <width>701</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Note: you need to restart XFB for some of the changes to take effect</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>140</y>
            <width>381</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Language:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbox_lang">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">selection-color: rgb(87, 118, 255);</string>
             </property>
             <item>
              <property name="text">
               <string>English</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Français</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Portugûes</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>711</width>
            <height>112</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="checkBox_disableSeekBar">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This wil disable the track's seek bar so that the user is not able to seek trougth the song&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Disable the Seek Bar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_disableVolume">
             <property name="text">
              <string>Disable the Volume Slider</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_Normalize_Soft">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This will boost the volume of the output softly for low and medium sounds. Will not boost when the volume of the track is already high.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Enable Soft Live Sound Normalization (experimental)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_fullScreen">
             <property name="text">
              <string>Start in FullScreen</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Database</string>
         </attribute>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>131</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Selected database:</string>
          </property>
         </widget>
         <widget class="QLabel" name="txt_selected_db">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>10</y>
            <width>451</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>[NO DATABASE LOADED]</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>291</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Database Version: 1.0a jellyfish</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>120</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Delete Music Table</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>80</y>
            <width>251</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 4);</string>
          </property>
          <property name="text">
           <string>Careful !</string>
          </property>
         </widget>
         <widget class="QPushButton" name="f_bt_del_jingles_table">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>160</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Delete Jingles Table</string>
          </property>
         </widget>
         <widget class="QPushButton" name="f_bt_del_pub_table">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>200</y>
            <width>241</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Delete Pub Table</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="recording">
         <attribute name="title">
          <string>Recording and Paths</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>30</y>
            <width>531</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Default recording device:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cboxRecDev">
             <property name="styleSheet">
              <string notr="true">selection-color: rgb(87, 118, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>170</y>
            <width>531</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Default recording path:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_savePath"/>
           </item>
           <item>
            <widget class="QToolButton" name="bt_browseSavePath">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>200</y>
            <width>531</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Default program path:  </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_programsPath"/>
           </item>
           <item>
            <widget class="QToolButton" name="bt_browse_programPath">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>61</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/icons/hardinfo.png</pixmap>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>60</y>
            <width>531</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Codec:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_codec">
             <property name="styleSheet">
              <string notr="true">selection-color: rgb(87, 118, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>90</y>
            <width>531</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Container:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_container">
             <property name="styleSheet">
              <string notr="true">selection-color: rgb(87, 118, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_19">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>160</y>
            <width>51</width>
            <height>51</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/icons/ic_menu_archive.png</pixmap>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget_2">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>230</y>
            <width>531</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Default music path:       </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_musicPath"/>
           </item>
           <item>
            <widget class="QToolButton" name="bt_browse_musicPath">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget_3">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>260</y>
            <width>531</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Default Jingle path:        </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_jinglePath"/>
           </item>
           <item>
            <widget class="QToolButton" name="bt_browse_jinglePath">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>130</y>
            <width>691</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Network</string>
         </attribute>
         <widget class="QCheckBox" name="cbox_enableNetworking">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>321</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Enable Networking</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>61</y>
            <width>351</width>
            <height>151</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Server URL:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="txt_server"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Port:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="txt_port"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>User:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="txt_user"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Password:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="txt_password">
             <property name="inputMethodHints">
              <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Role:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="cbox_role">
             <item>
              <property name="text">
               <string>Client</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Server</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>230</y>
            <width>631</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>FTP local temp folder</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_FTPlocalTempFolder"/>
           </item>
           <item>
            <widget class="QToolButton" name="bt_browseFTPlocalFolder">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>420</x>
            <y>60</y>
            <width>311</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Comunications hour:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="cboxComHour"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>263</y>
            <width>631</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>TakeOver temp folder:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_takeOverlocalTempFolder"/>
           </item>
           <item>
            <widget class="QToolButton" name="bt_browseTakeOverlocalFolder">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>System Resources</string>
         </attribute>
         <widget class="QPlainTextEdit" name="txt_terminal">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>74</y>
            <width>721</width>
            <height>201</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(59, 59, 59);
color: rgb(255, 255, 255);</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>721</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="bt_uname">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shows operating system information&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>uname</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bt_pwd">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Print working directory (in console)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>pwd</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bt_free">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show the amount of free RAM resources (in MB)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>free</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bt_df">
             <property name="toolTip">
              <string>Shows the free/used HD space</string>
             </property>
             <property name="text">
              <string>df</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>721</width>
            <height>27</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="bt_update_youtubedl">
             <property name="text">
              <string>youtube-dl troubleshoot</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bt_edit_settings">
             <property name="text">
              <string>Manually edit settings.conf</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Close (and save)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="bt_save_settings">
     <property name="text">
      <string>Save Settings</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Close (without saving)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
